mender-update/CMakeLists.txt

```diff
@@ -154,7 +154,9 @@ target_link_libraries(mender_update_cli_test PUBLIC
   gmock
 )
 target_compile_options(mender_update_cli_test PRIVATE ${PLATFORM_SPECIFIC_COMPILE_OPTIONS})
-gtest_discover_tests(mender_update_cli_test)
+# Use NO_PRETTY_VALUES to avoid very long byte strings in the output due to
+# parametrized tests that have binary objects as input.
+gtest_discover_tests(mender_update_cli_test NO_PRETTY_VALUES)
```

Comment:
Trust me, what it prints is just garbage. Structs turned into a raw byte stream with no structure. It's not really necessary because it still prints the name, so you can identify the test case.

Reply:
Or is this related to the `<<` operator you made? That would be prettier, obviously, but I think it's way too verbose to printed for every test.

Reply:
yes